src/components/GSAHeader.astro

```diff
@@ -31,10 +31,28 @@ import searchIcon from "../assets/images/usa-icons/search--white.svg"
             </button>
             <ul id="extended-nav-section-about" class="usa-nav__submenu">
               <li class="usa-nav__submenu-item">
-                <a href=""><span class="margin-x-1">&lt;Navigation link&gt;</span></a>
+                <a href="about/"><span class="margin-x-1">About GSA SmartPay</span></a>
```

Comment:
This component will get used on interior pages like `/faq/`. When that happens, will relative links break? (I think this will become `/faq/about/`) I think these will need to be absolute links to work globally on the site `/about/`. This will then need to have the base url added. The link with the logo in the header has an example.

Reply:
updated the code to added base url